ipython/bld.bat

```diff
@@ -5,4 +5,4 @@ python setup.py install
 if errorlevel 1 exit 1
 
 rd /s /q %SCRIPTS%
-rd /s /q %SP_DIR%\ipython-%PKG_VERSION%-py%PY_VER%.egg\share
```

Comment:
Ok, then please remove it :-) Sorry for the noise.

El 07/12/15 a las 11:10, Bruno Oliveira escribió:
>
> In ipython/bld.bat 
> <https://github.com/conda/conda-recipes/pull/500#discussion_r46839066>:
>
> > @@ -5,4 +5,4 @@ python setup.py install
> >  if errorlevel 1 exit 1
> >
> >  rd /s /q %SCRIPTS%
> > -rd /s /q %SP_DIR%\ipython-%PKG_VERSION%-py%PY_VER%.egg\share
>
> Same error:
>
> |rd /s /q 
> E:\Miniconda\envs\_build\Lib\site-packages\ipython-3.2.1-py2.7.egg The 
> system cannot find the file specified. Command failed: 
> C:\Windows\system32\cmd.exe /c call bld.bat |
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/conda/conda-recipes/pull/500/files#r46839066>.
>